.release-please-manifest.json

```diff
@@ -1,3 +1,3 @@
 {
-  ".": "2.5.6"
```

Comment:
This should not be manually edited. It's used by release please to find the previous version

Reply:
Done